bitnami/discourse/templates/deployment.yaml

```diff
@@ -99,11 +99,26 @@ spec:
           command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
           {{- else if .Values.discourse.command }}
           command: {{- include "common.tplvalues.render" (dict "value" .Values.discourse.command "context" $) | nindent 12 }}
+          {{- else }}
+          command:
+            - /bin/bash
           {{- end }}
           {{- if .Values.diagnosticMode.enabled }}
           args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
           {{- else if .Values.discourse.args }}
           args: {{- include "common.tplvalues.render" (dict "value" .Values.discourse.args "context" $) | nindent 12 }}
+          {{- else }}
+          args:
+            - -c
+            - |
+              {{- if .Values.discourse.plugins }}
+              pushd "/opt/bitnami/discourse" >/dev/null || exit
+              {{- range $plugin := .Values.discourse.plugins }}
+              RAILS_ENV=production bundle exec rake plugin:install repo={{ $plugin }}
+              {{- end }}
```

Comment:
Can you confirm that this works if you upgrade the chart, after having installed some plugins during the 1st initialization?

Reply:
Yes! It works when upgrading.

Plugins will be persisted at `/bitnami/discourse/plugins`, so even if the `discourse.plugins` value is only provided once, they will be persisted between upgrades.

I have tested the following scenarios:

- Install the `bitnami/discourse` chart with Oauth plugin, enable the plugin in the UI, upgrade the chart without providing the plugins value. The chart is upgraded successfully, the plugin and its settings are persisted.
- Install the `bitnami/discourse` chart with Oauth plugin, enable the plugin in the UI, upgrade the chart with the plugins value again. The chart is upgraded successfully, the plugin and its settings are also persisted.